elastic/kibana
# [Metric threshold rule] Adds new context variable for group by keys

## Summary

Closes https://github.com/elastic/kibana/issues/141496

Adds new context variable called `groupByKeys` which holds object of group by keys for Metric threshold rule.

### Manual Testing

1. Create Metric threshold rule with group by e.g. `host.name`, `host.network.name`
2. Define action template with `context.groupByKeys`
3. Wait for alerts to be generated
4. Notice the group by object to be included in the alert notification

<img width="883" alt="Screenshot 2022-11-21 at 12 27 20" src="https://user-images.githubusercontent.com/69037875/203039799-1465a8dc-cdf6-4adb-8fe2-9ced5833c3be.png">

### Checklist
